packages/good-design/src/theme/theme.tsx

```diff
@@ -8,12 +8,26 @@ export const theme = extendTheme({
     initialColorMode: "dark"
   },
   colors: {
-    text1: '#0D182D',
-    heading: '#42454A'
+    text1: "#0D182D",
+    heading: "#42454A",
+    gdBlueExample: {
```

Comment:
Try use semantic names as I did

Reply:
I did this because it's temp up until the designers provide us with our branding styles